src/Z3/Base/C.hsc

```diff
@@ -562,7 +562,8 @@ foreign import ccall unsafe "Z3_add_rec_def"
     z3_add_rec_def :: Ptr Z3_context
                    -> Ptr Z3_func_decl
                    -> CUInt
-                   -> Ptr Z3_ast_vector
+--                   -> Ptr Z3_ast_vector
```

Comment:
You can delete my wrong type instead of commenting it :smile:

Reply:
I was surprised it compiled with the bad type

Reply:
Yup, but I think that is an inherent property of the FFI.